pkg/didcomm/protocol/introduce/states.go

```diff
@@ -93,11 +89,11 @@ func (s *start) CanTransitionTo(next state) bool {
 	return false
 }
 
-func (s *start) ExecuteInbound(ctx internalContext, _ *metaData) (state, error) {
+func (s *start) ExecuteInbound(_ dispatcher.Outbound, _ *metaData) (state, error) {
 	return nil, errors.New("start ExecuteInbound: not implemented yet")
 }
```

Comment:
will these ever be implemented?

Reply:
I am not sure, we do not use it for now. But better to return an error. It will help us to understand where an error happens.

Reply:
sure. i just think the error text is misleading if this is not going to be implemented.

Reply:
Any specific suggestions?